ecosystem/typescript/sdk/src/plugins/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from "./token_client";
+export * from "./aptos_token";
```

Comment:
The name `aptos_token` is not quite distinguishable from `token_client`. People have no idea which they should use.

Reply:
I used `aptos_token` because this is the move module name. any suggestions for a name?